docs/language/learn-ql/writing-queries/debugging-queries.rst

```diff
@@ -0,0 +1,87 @@
+Query writing: common issues
```

Comment:
Should this specifically mention performance issues?

Reply:
At present it only mentions performance issues, so yes. But I'm not sure what other detail may end up in this topic. 
Can you think if anything else that should be mentioned?